spec/rails_helper.rb

```diff
@@ -52,3 +52,10 @@
 
   config.global_fixtures = :all
 end
+
+Shoulda::Matchers.configure do |config|
+  config.integrate do |with|
+    with.test_framework :rspec
+    with.library :rails
+  end
+end
```

Comment:
I have no strong feelings one way or the other, at this point. I think it's a good idea to keep an eye on this and consider taking action if the helper file starts getting out of hand. At the moment, though, it's 60 lines of mostly comments, so I'm OK to let it be